code/game/objects/items/weapons/defib.dm

```diff
@@ -67,16 +67,15 @@
 	else
 		..()
 
-/obj/item/defibrillator/MouseDrop()
-	if(ismob(src.loc))
-		if(!CanMouseDrop(src))
-			return
-		var/mob/M = src.loc
-		if(!M.unEquip(src))
-			return
-		src.add_fingerprint(usr)
-		M.put_in_hands(src)
-
+// what is this proc doing?
+/obj/item/defibrillator/handle_mouse_drop(var/atom/over, var/mob/user)
+	if(ismob(loc))
+		var/mob/M = loc
+		if(M.unEquip(src))
+			add_fingerprint(usr)
+			M.put_in_hands(src)
+			return TRUE
```

Comment:
Just noting that it's pretty confusing that truthy returns in this one mean "handled", but in the other one they mean "not handled."

Reply:
I could swap the result and checks I guess. Seems a bit more confusing to me, but it would be more consistent.

Reply:
Swapped.